CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Changelog
 
 ## Unreleased
-- Fixed an issue where *update-release-notes* and *doc-review* did not handle new content notes as expected.
+* Fixed an issue where *update-release-notes* and *doc-review* did not handle new content notes as expected.
```

Comment:
It shows as if I changed the whole line, but I only changed `-` to  `*` at the start